### PR TITLE
fix(docgen): support @values on classPropHandler

### PR DESCRIPTION
**Pull request by usei95**
#662

### CODE REVIEW

**Comment by elevatebart** [comment]
Can you add a unit test in the corresponding file? 
It makes maintenance easier.
Plus, when we want to refactor we can make sure than nothing breaks.

File: packages/vue-docgen-api/src/script-handlers/classPropHandler.ts [hunk 1]
```diff
@@ -53,6 +53,15 @@ export default async function classPropHandler(
...
 				}
 
+				if (propDescriptor.tags && propDescriptor.tags['values']) {
+					const description = ((propDescriptor.tags['values'][0] as any) as ParamTag).description
+					const choices = typeof description === 'string' ? description.split(',') : undefined
+					if (choices) {
+						propDescriptor.values = choices.map((v: string) => v.trim())
+					}
+					delete propDescriptor.tags['values']
+				}
```
Comment (elevatebart) [inline packages/vue-docgen-api/src/script-handlers/classPropHandler.ts]: Have you tried extracting this into a function? 
I would probably host it in the `propHandler.ts` file.

**Comment by elevatebart** [comment]
I created the function and added some tests to propHandler. 
I forgot to create them myself. How could I ask someone else to test?

You can now use the function and write the tests ;-)

Thank you so much

**Comment by usei95** [comment]
> I created the function and added some tests to propHandler.
> I forgot to create them myself. How could I ask someone else to test?
> 
> You can now use the function and write the tests ;-)
> 
> Thank you so much

Thanks, i added test and updated code its now using extractValuesFromTags in propHandler.

**Pull request merged**